packages/terra-functional-testing/tests/wdio/viewport-helpers-spec.js

```diff
@@ -1,24 +0,0 @@
-Terra.describeViewports('Viewport Helpers', ['small', 'large'], () => {
-  it('creates tiny and huge screens', () => {
-    browser.url('/compare.html');
-    // Terra.validates.screenshot();
```

Comment:
This was initially created to test capturing screenshots and viewports. We have since added the terra-validates-spec.js wdio test and getViewports.test.js jest tests so this is no longer appropriate or needed.